alembic/versions/752c338c6328_seed_db.py

```diff
@@ -12,25 +12,26 @@
 
 
 # revision identifiers, used by Alembic.
-revision = 'dc382319dd84'
+revision = '752c338c6328'
 down_revision = '8c089a511ab7'
 branch_labels = None
 depends_on = None
 
 
 def upgrade():
-    plants_table = table('plant',
+    plants_table = table('plants',
+        column('id', Integer),
         column('name', String),
         column('plant_type', String),
         column('image', Unicode),
         column('lighting', String),
         column('water_frequency', Integer),
         column('harvest_time', Integer),
         column('root_depth', Integer),
-        column('annual', Boolean)
-        )
+        column('annual', Boolean),
+    )
 
-    op.bulk_insert('plants_table',
+    op.bulk_insert(plants_table,
```

Comment:
Really nice work figuring out this bulk inserts.  Who knew it would all come down to what was in quotes and what wasn't!